docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
     image: itkdev/php8.1-fpm:latest
```

Comment:
See comment about php version

Reply:
They are using php 8.2 them self: https://github.com/tiredofit/docker-freescout/blob/main/Dockerfile#L1

https://endoflife.date/php

Reply:
This follows the https://github.com/itk-dev/devops_itkdev-docker/blob/develop/templates/symfony-6/docker-compose.yml but is overridden in docker-compose.server.override.yml

Reply:
okay